state/convert.go

```diff
@@ -169,6 +169,8 @@ func deriveEventsHash(spork *config.Spork, events []flowEvent) flow.Identifier {
 		return deriveEventsHashV1(events)
 	case 2, 3:
 		return deriveEventsHashV2(events)
+	case 4:
+		return deriveEventsHashV3(events)
```

Comment:
Please name the function in the version they were introduced, i.e. `deriveEventsHashV4` — thanks!